src/view/application/Application.js

```diff
@@ -0,0 +1,5 @@
+const Application = () => {
+  return <div> hello word</div>;
```

Comment:
Apply a Tailwind class in here so we know it works.

Reply:
done